### DISCUSSION TITLE
myMPD has now 200 GitHub stars

### DISCUSSION THREAD

**Discussion by jcorporation** [jcorporation/myMPD#756] [General]
I'm pouring a virtual round of champagne!

myMPD started in 2018 with small patches to enable albumart for ympd and then has evolved to a full blown MPD client.

**Comment by john-** [comment]
Congratulations!  You have a clear vision of how to improve myMPD and the technical ability to pull it off.  Early on in development I got comfortable that any changes you made (many pretty big) were going to be for the better and be well executed.

Great work and thanks!

**Comment by Emilot** [comment]
Nothing else than clear:

**Awesome** **work JC**

Thank you very much!

**Comment by sevmonster** [comment]
myMPD is my primary music client. I used to use foobar2000 but now use MPD and myMPD almost exclusively.

It's a wonderful product, thanks for all your hard work.

**Comment by jcorporation** [comment]
Reached 400 stars, doubled the star within two years. It seems this project is popular :)

**Reply by john-** [reply]
Great to see so many people appreciate it!
